README.md

```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
+* The `click` action waits for the element to be visible by default.
+* User agent and device emulation is set up consistently on the `BrowserContext` level to enable multi-page scenarios.
+* Playwright supports XPath as a first-class selector and adds [custom selector engines](selectors.md).
 
-We recognize WebDriver as a universal standard for the web automation and testing. At the same time we were excited to see Puppeteer affect the WebDriver agenda, steer it towards the bi-directional communication channel, etc. We hope that Playwright can take it further and pioneer support for numerous PWA features across the browsers as they emerge:
+All the changes and improvements would require breaking changes to the Puppeteer API, and so we chose to start with a clean slate instead. Due to the similarity of the concepts and the APIs, migration between the two should be a mechanical task.
 
-- [*capabilities*] With Playwright, we aim at providing a more capable driver, including support for [mobile viewports](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag), [touch](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Using_Touch_Events), [web](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) & [service workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), [geolocation](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API), [csp](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP), [cookie policies](https://web.dev/samesite-cookies-explained/), [permissions](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API), [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility), etc.
+#### Q: What about [Selenium WebDriver](https://www.w3.org/TR/webdriver/)?
 
-- [*ergonomics*] We continue the trend set with Puppeteer and provide ergonomically-sound APIs for frames, workers, handles, etc.
+We recognize WebDriver as a universal standard for the web automation and testing. At the same time we were excited to see Puppeteer affect the WebDriver agenda and steer it towards a bi-directional channel. We hope that Playwright can take it further and pioneer support for numerous PWA features across the browsers as they emerge.
 
-- [*reliability*] With Playwright, we encourage `setTimeout`-free automation. The notion of the wall time is incompatible with the operation in the cloud / CI. It is a major source of flakiness and pain and we would like to provide an alternative. With that, Playwright aims at providing sufficient amount of events based on the browser instrumentation to make it possible.
+- **Reliability**: With Playwright, we encourage `setTimeout`-free automation. The notion of the wall time is incompatible with cloud operation and is a major source of flakiness. Playwright provides sufficient amount of events based on the browser instrumentation to eliminate flakiness.

```

Comment:
Again, re-iterate **Capable** and **Reliable** ? Also, with the top-level items I feel like we repeat each other here.

Reply:
thanks for the review! updating the pitch to be more clear and removing redundancies with the FAQs